deploy/multi-platform/README.md

```diff
@@ -0,0 +1,46 @@
+## Wavefront Collector images for multi-platform

```

Comment:
Can you clarify what this is used for? Not clear why we need this.

Reply:
The manifest tool is to create a common image name for multiple platforms.
The collector image can be pulled using the same name `wavefronthq/wavefront-kubernetes-collector` for both Linux & Windows platform.
Reference doc: https://wavefront.atlassian.net/l/c/uk0Q21up